subworkflows/nf-core/qc_markduplicates.nf

```diff
@@ -88,12 +79,11 @@ workflow QC_MARKDUPLICATES {
     //if !skip_markduplicates, then QC tools are run on duplicate marked bams
     //After bamqc finishes, convert to cram for further analysis
     qualimap_bamqc = Channel.empty()
-    if(!skip_bamqc){
+    if (!skip_bamqc && !skip_markduplicates) {
```

Comment:
should this not be run on the mapped bams if no duplicate marking is done?

Reply:
I'll need to do more testing and add them to CI to have all scenario covered.
I figured it was a big enough PR already ;-)

Reply:
ok then let's add a TODO here so it won't sneak by unnoticed later

Reply:
```suggestion
    if (!skip_bamqc && !skip_markduplicates) {
    //TODO: after adding CI tests, allow bamqc on mapped bams if no duplicate marking is done
```